### ISSUE TITLE
Versioning or tagging support

### ISSUE THREAD

**Issue by zladuric** [GetPublii/Publii#760]
I'm trying to find, unsuccessfully, some way to support versioning or tagging. What I mean is a way to make a snapshot/freeze the website, maybe tag it with a timestamp, SHA or preferably a custom tag.

That way I can work on a site, deploy it and set the current snapshot as a "prod" tag or "prod" version. Then I can go work on a new article, and if I need to fix a typo in older content, I can switch back to the other version momentarily.

Basically, hide git behind this awesomely simple-to-use tool for non-technical users :)

I haven't seen such a feature here in the issues or on the roadmap. If it doesn't exist, can it be planned? If yes, I'm offering help working on it :)

**Comment by IanMayo** [comment]
It may be a challenge for Publii, since the content is stored in a single SQLite database file (`sites/name/input/db.sqlite`).

I recognise the value in what you're proposing, but it may require a fundamentally different approach to the app.

**Comment by dziudek** [comment]
I think it looks like too complicated feature for the idea of Publii. You can always create a backup of your website if you need a snapshot of your website from the specific moment :)
